install/readme.md

```diff
@@ -18,13 +18,23 @@ export HOST_IP=<your_host_ip> # Replace with your actual host IP
 echo $HOST_IP
 ```
 
-- Export the desired SODA Release version to the environment variable
+- Export the desired Dashboard Release version to the environment variable
 
 ```sh
 export DASHBOARD_RELEASE_VERSION=v1.7.0 # Replace with SODA release version or docker tag for dashboard.
 echo $DASHBOARD_RELEASE_VERSION
 ```
 
+- Export the following environement variables for SRM Toolchain
+
+```sh
+export PROMETHEUS_IMAGE_TAG=v2.23.0
+export SODA_PROMETHEUS_PORT=9090
+export ALERTMANAGER_IMAGE_TAG=v0.21.0
+export SODA_ALERTMANAGER_PORT=9093
+export GRAFANA_IMAGE_TAG=7.3.5
+export SODA_GRAFANA_PORT=3000
```

Comment:
All parameters in SODA installer ansible/group_vars/srm_toolschain.yml needs to be taken as input for the script

Reply:
Yes. These are to be exported as environment variable. The env variable is then exposed to the docker container as well.